parser-typechecker/src/Unison/TermPrinter.hs

```diff
@@ -644,6 +646,7 @@ suffixCounterTerm n = \case
 suffixCounterType :: Var v => PrettyPrintEnv -> Type v a -> PrintAnnotation
 suffixCounterType n = \case
     Type.Var' v -> countHQ $ HQ.unsafeFromVar v
+    Type.Ref' r | noImportRefs r || r == Type.vectorRef -> mempty

```

Comment:
Why the check on `vectorRef` here but nothing for terms/patterns?

Reply:
There’s no list/vector/sequence constructor. Doesn’t hurt to check in both spots though if you want to just add to noImportRefs function.